xbmc/android/activity/XBMCApp.cpp

```diff
@@ -299,6 +301,38 @@ bool CXBMCApp::EnableWakeLock(bool on)
   return true;
 }
 
+bool CXBMCApp::EnableAudioFocus(bool on)
```

Comment:
`EnableWakeLock` is slightly different since it has one path that is taken regardless of the boolean parameter.